lib/modules/apostrophe-assets/public/js/always.js

```diff
@@ -172,6 +172,14 @@ _.extend(apos, {
       });
     }
   },
+  
+  enableHtmlPageId: function() {
```

Comment:
Move this to user.js, it should not be forced on browserland for logged out users